dns/README.rst

```diff
@@ -30,7 +30,7 @@ In order to use this library, you first need to go through the following steps:
 
 .. _Select or create a Cloud Platform project.: https://console.cloud.google.com/project
 .. _Enable billing for your project.: https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project
-.. _Enable the Google Cloud Datastore API.:  https://cloud.google.com/datastore
+.. _Enable the Google Cloud Datastore API.:  https://cloud.google.com/dns
```

Comment:
Thanks for the fix! It still says "Datastore" in the text, `"Enable the Google Cloud Datastore API."`